src/test_fixture/world.rs

```diff
@@ -237,6 +237,7 @@ where
         <[_; 3]>::try_from(output).unwrap()
     }
 
+    #[allow(unused_must_use)]
```

Comment:
Why not just mark this must_use ?

Reply:
Its funny, there is a test_fixture validate which doesn't return anything. The compiler is confusing test_fixture::validate with MaliciousValidatos::validate.